### PR TITLE
Allow customers to access extension

### PR DESCRIPTION
**Pull request by laurisb**

### CODE REVIEW

File: src/plib/controllers/IndexController.php [hunk 1]
```diff
@@ -5,14 +5,26 @@
 
 class IndexController extends pm_Controller_Action
 {
+    private $client;
     private $currentPath = '/';
+    private $basePath = '/';
 
-    protected $_accessLevel = 'admin';
+    protected $_accessLevel = ['admin', 'client'];
```
Comment (xgin) [inline src/plib/controllers/IndexController.php]: add also 'reseller'

File: src/plib/library/Helper.php [hunk 2]
```diff
@@ -26,21 +26,29 @@ public static function cleanPath($path)
...
     public static function getDiskspaceUsage($path)
     {
-        $list = [];
-        $result = \pm_ApiCli::callSbin('diskspace_usage.sh', [$path]);
+        $client = \pm_Session::getClient();
+
+        if ($client->isAdmin()) {
+            $result = \pm_ApiCli::callSbin('diskspace_usage.sh', [$path]);
+        } else {
+            $username = $client->getLogin();
```
Comment (xgin) [inline src/plib/library/Helper.php]: `getSysUserLogin` from current domain

File: src/plib/controllers/IndexController.php (hunk 1 above)
Comment (laurisb) [reply src/plib/controllers/IndexController.php]: Done

File: src/plib/library/Helper.php (hunk 2 above)
Comment (laurisb) [reply src/plib/library/Helper.php]: Done

**Pull request merged**